### PR TITLE
More accurate jfrog setup to prevent skipping artifactoryDeploy for some subprojects

### PR DESCRIPTION
**Pull request by fishinitself**
This fixes the following bug - in case of particular gradle project with a few subprojects, one of subprojects didn't deploy its results to artifactory. This was caused by late applying and configuring com.jfrog.artifactory to subprojects

### CODE REVIEW

File: ft/src/test/resources/publish/multi-module-regression/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-all.zip
```
Comment (pgorbachev) [inline ft/src/test/resources/publish/multi-module-regression/gradle/wrapper/gradle-wrapper.properties:3]: 8.11.1

File: ft/src/test/resources/publish/multi-module-regression/rmtest-core/src/main/java/org/octopusden/rmtest/core/utils/StringPair.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package org.octopusden.rmtest.core.utils;
+
+import java.util.Map;
+
+public class StringPair implements Map.Entry<String, String> {
```
Comment (pgorbachev) [inline ft/src/test/resources/publish/multi-module-regression/rmtest-core/src/main/java/org/octopusden/rmtest/core/utils/StringPair.java:5]: Для чего нужен этот класс?

**Review by pgorbachev — CHANGES_REQUESTED** [review]
Предлагаю почистить тестовый проект

File: ft/src/test/resources/publish/multi-module-regression/rmtest-core/src/main/java/org/octopusden/rmtest/core/utils/StringPair.java (hunk 2 above)
Comment (fishinitself) [reply ft/src/test/resources/publish/multi-module-regression/rmtest-core/src/main/java/org/octopusden/rmtest/core/utils/StringPair.java]: Да куда уже меньше... Ну ок, упростил

**Review by ozonophore — APPROVED** [review]

**Review by pgorbachev — APPROVED** [review]

**Review by CamaradeRoman — APPROVED** [review]

**Pull request merged**